package.json

```diff
@@ -5,6 +5,9 @@
   "main": "index.js",
   "author": "Andrew Balmos <andrew@balmos.org> (https://github.com/abalmos)",
   "license": "Apache 2.0",
+  "scripts": {
+    "test": "./istantestd.sh"
```

Comment:
I have no strong feeling either way. Would there be a need to run the test but not get coverage?